PINRemoteImage/PINRemoteImage.xcodeproj/project.xcworkspace/xcshareddata/PINRemoteImage.xcscmblueprint

```diff
@@ -0,0 +1,30 @@
+{
+  "DVTSourceControlWorkspaceBlueprintPrimaryRemoteRepositoryKey" : "9B1DC435B89529078DE1FAE4D39ED8D2F4E74032",
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyRepositoryLocationsKey" : {
+
+  },
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyStatesKey" : {
+    "9B1DC435B89529078DE1FAE4D39ED8D2F4E74032" : 0,
+    "44B36422A7A7C3AD3DE414BC27AA89E3C0009BBA" : 0
+  },
+  "DVTSourceControlWorkspaceBlueprintIdentifierKey" : "55C0E8B1-B64A-4875-83E1-ABB927D232FA",
+  "DVTSourceControlWorkspaceBlueprintWorkingCopyPathsKey" : {
+    "9B1DC435B89529078DE1FAE4D39ED8D2F4E74032" : "PINRemoteImage\/",
+    "44B36422A7A7C3AD3DE414BC27AA89E3C0009BBA" : "PINCache\/"
+  },
+  "DVTSourceControlWorkspaceBlueprintNameKey" : "PINRemoteImage",
+  "DVTSourceControlWorkspaceBlueprintVersion" : 204,
+  "DVTSourceControlWorkspaceBlueprintRelativePathToProjectKey" : "PINRemoteImage\/PINRemoteImage.xcodeproj",
+  "DVTSourceControlWorkspaceBlueprintRemoteRepositoriesKey" : [
+    {
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "https:\/\/github.com\/pinterest\/PINCache.git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryIdentifierKey" : "44B36422A7A7C3AD3DE414BC27AA89E3C0009BBA"
+    },
+    {
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "github.com:foursquare\/PINRemoteImage.git",
```

Comment:
Ah yeah, that is probably a mistake. You need to check in your shared framework scheme (which is in xcshareddata) so that Carthage can find and build it, but not this blueprint file I don't think. I will fix.